<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PdfMenu</class>
 <widget class="QWidget" name="PdfMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>PDF处理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mergePdfBtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>170</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>PDF合并</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mergedistUnitNoBtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>170</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>按不动产单元号合并</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
